src/renderer/components/TopNavbar.js

```diff
@@ -109,6 +110,10 @@ class TopNavbar extends React.Component {
             window.history.back()
         }
 
+        function goToClip() {
+            navigator.clipboard.readText().then(clipText => {window.location.replace(clipText)});
```

Comment:
Make sure to only use the location hash, not the full path.

Reply:
That is what I used from the onset, refer to the `windowLocationUrl` variable